backend/app/api/api_v1/routers/lookups.py

```diff
@@ -16,15 +24,44 @@ def table_to_json(
     json_out = []
 
     for row in db.query(table).all():
-        row_object = {}
-        for col in row.__table__.columns:
-            row_object[col.name] = getattr(row, col.name)
-
+        row_object = {
+            col.name: getattr(row, col.name) for col in row.__table__.columns
+        }
         json_out.append(row_object)
 
     return json_out
 
 
+def tree_table_to_json(
+    table: Base,
+    db: SessionLocal,  # type: ignore
+) -> List[Dict]:
+    json_out = []
+    child_list_map: Dict[int, Any] = {}
+
+    for row in db.query(table).all():
+        row_object = {
+            col.name: getattr(row, col.name) for col in row.__table__.columns
```

Comment:
If `getattr(row, col.name)` returns a date, it won't serialise. Maybe check for date/datetime instance, and use `.isoformat()`?
Alternatively, I'm doing something similar in a previous PR, and deferring the decision to use something like Marshmallow a bit later.